pkg/authorization/util/util.go

```diff
@@ -137,3 +138,17 @@ type AuthorizationStorage struct {
 	ClusterRole          clusterrole.Storage
 	ClusterRoleBinding   clusterrolebinding.Storage
 }
+
+func GetAuthorizationProxy(rbacclient rbacinternal.RbacInterface) (*AuthorizationStorage, error) {
```

Comment:
TODO probably want to move need proxy funcs out so they do not need to confirm to the old interfaces.